backends-velox/src/main/java/io/glutenproject/vectorized/ArrowBufBuilder.java

```diff
@@ -29,9 +31,9 @@ public class ArrowBufBuilder {
    * Create an instance for ArrowBufBuilder.
    *
    * @param nativeInstanceId native ArrowBuf holder.
-   * @param memoryAddress native ArrowBuf data addr.
-   * @param size ArrowBuf size.
-   * @param capacity ArrowBuf rowNums.
+   * @param memoryAddress    native ArrowBuf data addr.
+   * @param size             ArrowBuf size.
+   * @param capacity         ArrowBuf rowNums.
```

Comment:
The same

Reply:
Change it to compact mode in new added commit